naloge/bfs.tex

```diff
@@ -10,6 +11,112 @@
 \caption{Graf za nalogi~\nal in~\nal[dfs].}
 \end{slika}
 \end{vprasanje}
+
 \begin{odgovor}
+ALGORITEM:\\
+\underline{VHOD}:  graf G, vozlišče $s \in V$\\
+\underline{IZHOD}: razdalje $d_G(s, v)$ za vsak $v \in V$\\
+def bfs(G, s):\\
+	\phantom{x}\hspace{3ex}za vsak $v \in V(G)$:\\
+		\phantom{x}\hspace{6ex}razdalja[v] = $\infty$\\
+	\phantom{x}\hspace{3ex}razdalja[s] = 0\\
+	\phantom{x}\hspace{3ex}Q = [s]\\
+	\phantom{x}\hspace{3ex}dokler Q ni prazna:\\
+		\phantom{x}\hspace{6ex}u = odstrani-prvega(Q)\\
+		\phantom{x}\hspace{6ex}za vsak $(u, v) \in E(G)$:\\
+			\phantom{x}\hspace{9ex}če razdalja[v] = $\infty$:\\
+				\phantom{x}\hspace{12ex}razdalja[v] = razdalja[u] + 1\\
+				\phantom{x}\hspace{12ex}vstavi(Q, v)\\
+\begin{flushleft}
```

Comment:
Svetujem, da se okolij `flushleft` znebiš, saj, kolikor vidim, ne vplivajo na izgled (pa tudi drugod se ne uporabljajo, tako da bi na ta način kvečjemu odstopali od siceršnje oblike). Nasploh naj bo direktnega oblikovanja čim manj.

Reply:
Hvala za popravek. Sem pa opazila, da se brez uporabe ``flushleft`` prva vrstica vsakega novega odstavka zamakne, kar ni izgledalo lepo, zato sem to dodala. Želite, da vseeno odstranim?

Reply:
Kar odstrani. Do zamika pride v začetkih odstavkov, torej po praznih vrsticah v izvorni kodi. Če besedilo nadaljuješ brez prazne vrstice, ne bo prišlo do zamikov (če bi rada vseeno imela v kodi skoraj prazno vrstico, lahko postaviš `%`, torej vrstico zakomentiraš, da ne bo štela za prazno).

Reply:
Hvala, bom popravila.